src/assets/src/components/queue.tsx

```diff
@@ -98,6 +98,7 @@ function QueueAttendingNotJoined(props: QueueAttendingProps) {
             <ul>
                 <li>Number of people currently in line: <strong>{props.queue.line_length}</strong></li>
                 <li>You are not in the meeting queue yet</li>
+                <li>In-person meetings will take place at: <strong>{props.queue.physLocation}</strong></li>
```

Comment:
```suggestion
                <li>In-person meetings will take place at: <strong>{props.queue.phys_location}</strong></li>
```
To work with other changes.